reposilite/reposilite.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "reposilite",
+  "display": "Reposilite",
+  "type": "other",
+  "install": [
+    {
+      "type": "javadl",
+      "version": "${javaversion}"
+    },
+    {
+      "files": [
+        "https://maven.reposilite.com/releases/com/reposilite/reposilite/${version}/reposilite-${version}-all.jar"
+      ],
+      "type": "download"
+    }
+  ],
+  "run": {
+    "stop": "stop",
+    "command": "java -Xmx${memory}M -jar reposilite-${version}-all.jar --port ${port} --token ${token_name}:${secret}",
```

Comment:
Since using javadl, need to also use java${javaversion} here, so it uses the correct binary